src/instructor/components/AddAssignment/Index.vue

```diff
@@ -15,7 +15,7 @@
         </el-col>
         <el-col :span="18" style="float: right;">
             <v-state v-if="change" v-on:changeState="changeState"></v-state>
-            <v-add v-if="!change" v-on:goBack="goBack" style="margin-right: 15%"></v-add>
+            <v-add v-else v-on:goBack="goBack" style="margin-right: 15%"></v-add>
```

Comment:
Tags or attributes names starting with 'v' indicates that they belong to Vue.js. Your custom element should have a name to distinguish from HTML native elements. (e.g. do not use <aside> or <v-aside>, use <assignment-aside> or <qlt-aside>)

Reply:
do not use 'aside' (HTML5 native tag) or 'v-aside', try 'assignment-aside' or 'qlt-aside'

Reply:
save it to reconstitution